### ISSUE TITLE
Extranet responsive email designs

### ISSUE THREAD

**Issue by Dnpizarro** [cfpb/email-templates#5]
**Extranet email type 1**

| 600 | 320 |
| --- | :-: |
| ![cfpb_extranet_responsive_email_r2_extranet email type 1](https://cloud.githubusercontent.com/assets/6852363/6049490/10114990-ac86-11e4-9766-a2e736cbfbe4.png) | ![cfpb_extranet_responsive_email_r2_extranet email type 1 - mobile](https://cloud.githubusercontent.com/assets/6852363/6049493/19781c66-ac86-11e4-8a2d-6dfb2a5f5baa.png) |

**Extranet email type 2**

| 600 | 320 |
| --- | :-: |
| ![cfpb_extranet_responsive_email_r2_extranet email type 2](https://cloud.githubusercontent.com/assets/6852363/6049502/28a90060-ac86-11e4-90c2-c5a196a12ec4.png) | ![cfpb_extranet_responsive_email_r2_extranet email type 2 - mobile copy](https://cloud.githubusercontent.com/assets/6852363/6049504/2dc6f688-ac86-11e4-8658-7fa75e4e98a7.png) |

**Extranet email type 3**

| 600 | 320 |
| --- | :-: |
| ![cfpb_extranet_responsive_email_r2_extranet email type 3](https://cloud.githubusercontent.com/assets/6852363/6049508/3827ba68-ac86-11e4-8ade-91af64a7786c.png) | ![cfpb_extranet_responsive_email_r2_extranet email type 3 - mobile](https://cloud.githubusercontent.com/assets/6852363/6049512/3f26fcc0-ac86-11e4-982d-eaa5f47bfc17.png) |

**Comment by Dnpizarro** [comment]
So I just realized this but for the Extranet email type 1, the way I'm styling the links is implying that the user is already within our website, which isn't true. So, I'm going to have to reconsider how those links get styled. Maybe I'll just make them regular links. I'll post new mock-ups soon.

**Comment by Scotchester** [comment]
Could you elaborate? I don't follow.

**Comment by Dnpizarro** [comment]
I'm trying to address what would be the appropriate styling for the link depending on where the link is taking the user. 

In this example, as its being used in FJ, would suggest that's it taking you to another page on our website:
![screen shot 2015-02-04 at 4 58 03 pm](https://cloud.githubusercontent.com/assets/6852363/6050688/b89597fc-ac8f-11e4-8ffd-ea221ecae70d.png)

I'm looking at whether one of these other examples would make more sense:
![screen shot 2015-02-04 at 4 58 16 pm](https://cloud.githubusercontent.com/assets/6852363/6050707/da6b8f80-ac8f-11e4-94c9-0e5c3ae05972.png)

**Comment by Scotchester** [comment]
Ahah. Well, since they are all taking you out of an email client to a website, I don't know how much it matters. I'd vote for sticking with the right arrows.

**Comment by kurzn** [comment]
I totally agree with styling links differently to satisfy user expectations (open a new site, etc). But I do think that in the context of an email, every link is going to take the user out of the email itself. 

So given this, and the desire to keep things simple and not add additional weight or emphasis, I would suggest leaving them as just plain text links.

**Comment by netplusdesign** [comment]
@kurzn :+1:

**Comment by Dnpizarro** [comment]
Sounds good to me!
